packages/mobile/src/account/saga.ts

```diff
@@ -82,6 +84,28 @@ function* initializeAccount() {
   }
 }
 
+export function* watchDailyLimit() {
+  const account = yield call(getConnectedAccount)
```

Comment:
Do we actually need the account to be "connected"? Seems like this has nothing to do with Geth so probably better to just use `getAccount` right?

Reply:
Yup